test_settings.py

```diff
@@ -0,0 +1,3 @@
+"""Test settings for examquestionbank XBlock."""
```

Comment:
It is a good practice to have a folder called settings, with common, production, and test settings. It is a practice for Django settings and Open edX. 

You can find examples of this: https://github.com/eduNEXT/xblock-mindmap/tree/main/mindmap/settings

You'll also need to change the tox.ini to use the settings.